src/sign.ml

```diff
@@ -162,15 +162,17 @@ let unlink : t -> unit = fun sign ->
 (** [add_symbol sign mode name a] creates a fresh symbol with the name  [name]
     (which should not already be used in [sign]) and with the type [a], in the
     signature [sign]. The created symbol is also returned. *)
-let add_symbol : t -> sym_mode -> strloc -> term -> sym = fun sign mode s a ->
+let add_symbol : t -> sym_mode -> strloc -> term -> bool list -> sym =
+  fun sign mode s a implicits ->
```

Comment:
Add two spaces before `fun` (this is the convention used everywhere I think).

Also, you can rename `mode` into `sym_mode` and `implicits` into `sym_implicits` to save one line in the definition of `sym` (you can remove `= implicits` and ` = mode`, since a variable with the same name as the field is in scope).

Reply:
Done.